### ISSUE TITLE
Give us a ReadMe

### ISSUE THREAD

**Issue by Ciaran97** [Code-Bullet/Jump-King#8]
The lack of a ReadMe is indicative of your regular Youtube upload schedule. It does not exist

**Comment by LunaticBFF57** [comment]
I don't think he needs it as he has a direct page to play the game he "_created_".

**Comment by Xses-1** [comment]
Not really. p5 needs to run on the server, otherwise, the game doesn't load. That was certainly the case in my situation and it was the same with his webpage.

**Comment by Xses-1** [comment]
Here you go:
https://github.com/Xses-1/ReadME-for-Jump-King
